systems.py

```diff
@@ -5,34 +5,43 @@
 
 class Systems():
   
-  # Creates and initializes all of the controllers
+  """ Creates and initializes all of the controllers
+  """
   def __init__(self):
     self.controllers = [];
-    controllers[0] = MotorController()
-    controllers[1] = BatteryController()
-    controllers[2] = ClimateController()
-    controllers[3] = SensorController()
-    controllers[4] = BackupController()
+    self.controllers.append(MotorController())
+    self.controllers.append(BatteryController())
+    self.controllers.append(ClimateController())
+    self.controllers.append(SensorController())
+    self.controllers.append(BackupController())
 
-  # Finds and returns the value of the specified variable in a certain controller
+  """ Finds and returns the value of the specified variable in a certain controller
+  """
   def get(name: str) -> any:
     for i in controllers:
       value = controllers[i].getVariable(controllers[i], name)
-      return value
+      if value is not None:
+        return value
+    return None
```

Comment:
Take a look at the file controller.py as `getVariable()` will raise an exception if the variable doesn't exist, it will not just return None. Also, when calling member functions, although `self` is listed as the first parameter in the declaration, it is an implied argument and need not be passed when called. In this case, it would look like `getVariable(name)`.